analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TestClassShouldHaveTestMethod.NUnit3.cs

```diff
@@ -38,4 +38,16 @@ public class Fixture3 // Noncompliant
         [FakeAttribute]
         public void Baz() { }
     }
+
+    [TestFixture]
+    public class A // Noncompliant, FP - In NUnit this scenario is valid. See: https://github.com/SonarSource/sonar-dotnet/issues/5732
```

Comment:
Shouldn't we create issue on NUnit instead? 😆

Reply:
The `[TestFixture]` is more or less obsolete in NUnit. Just having the right test attributes (those who implement `IImplyFixture`, including `[Test]`) make that `[TestFixture]` is not required at all.